.github/workflows/main.yml

```diff
@@ -9,9 +9,12 @@ jobs:
 
     steps:
       - name: Select Xcode
-        run: sudo xcode-select -s /Applications/Xcode_11.3.1.app
+        run: sudo xcode-select -s /Applications/Xcode_11.4.1.app
+
+      - name: Install SwiftLint
+        run: brew install swiftlint
 
       - uses: actions/checkout@v1
 
       - name: Build
-        run: xcodebuild -scheme DarkModeKit -destination generic/platform=iOS -configuration Release | xcpretty -s
+        run: xcodebuild -scheme FluentDarkModeKit -destination generic/platform=iOS -configuration Release GCC_TREAT_WARNINGS_AS_ERRORS="YES" SWIFT_TREAT_WARNINGS_AS_ERRORS="YES" | xcpretty -s && exit ${PIPESTATUS[0]}
```

Comment:
Just curious: Why we need the `exit` at the end?

Reply:
See usage section of https://github.com/xcpretty/xcpretty

CI needs exit code to determine if the command has succeeded or not, xcpretty eats the exit code from xcodebuild and returns success even if xcodebuild failed. If you check a bit in the CI logs in previous commits (with wrong target `DarkModeKit ` specified), they all failed silently 👀